src/rest_api.py

```diff
@@ -35,3 +36,14 @@ class Feedback(Resource):
     def post(self):  # pylint: disable=no-self-use
         """Post feedback for an event."""
         return feedback(api.payload)
+
+
+@api.route('/api/v1/graph', methods=['POST'])
+class GraphQuery(Resource):
+    """Abstracts REST end-point routers."""
+
+    @api.doc(False)
+    def post(self):
+        """Endpoint to query graph db properties."""
+        response = GraphPassThrough().fetch_nodes(api.payload)
```

Comment:
You should consider creating a request model for this like existing code.

Reply:
For this request we are not exposing using api docs so i believe its not require. (api.docs(False))

Reply:
Model is not always for doc, it is easy to validate the user input with the help of request models with constraints. I would recommend to use request model regardless of whether we need swagger doc for that.

Reply:
Added request model